project/AppDependencies.scala

```diff
@@ -6,25 +6,25 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28"     % "5.3.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "0.64.0-play-28",
-    "uk.gov.hmrc"             %% "play-frontend-govuk"            % "0.71.0-play-28",
-    "uk.gov.hmrc"             %% "play-language"                  % "5.0.0-play-28",
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "0.53.0",
-    "uk.gov.hmrc"             %% "play-conditional-form-mapping"  % "1.9.0-play-28",
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28"     % "5.17.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"             % "1.31.0-play-28",
+    "uk.gov.hmrc"             %% "play-frontend-govuk"            % "2.0.0-play-28",
+    "uk.gov.hmrc"             %% "play-language"                  % "5.1.0-play-28",
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-28"             % "0.58.0",
+    "uk.gov.hmrc"             %% "play-conditional-form-mapping"  % "1.10.0-play-28",
     "uk.gov.hmrc"             %% "emailaddress"                   % "3.5.0"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"   % "5.3.0"            % "test, it",
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-28"  % "0.50.0"           % "test, it",
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"   % "5.17.0"            % "test, it",
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-28"  % "0.58.0"           % "test, it",
     "org.scalatest"           %% "scalatest"                % "3.0.9"            % "test, it",
     "org.jsoup"               %  "jsoup"                    % "1.13.1"           % "test, it",
     "com.typesafe.play"       %% "play-test"                % PlayVersion.current  % "test, it",
     "org.scalatestplus.play"  %% "scalatestplus-play"       % "5.1.0"            % "test, it",
     "com.vladsch.flexmark"     %  "flexmark-all"              % "0.36.8"           % "test, it",
```

Comment:
Can remove flexmark-all dependency. Is included as part of bootstrap-test-play 5.14.0+